packages/devtools_app/lib/src/shared/table/table_data.dart

```diff
@@ -129,14 +130,21 @@ mixin PinnableListEntry {
 /// will be drawn between groups and an additional header row will be added to
 /// the table to display the column group titles.
 class ColumnGroup {
-  ColumnGroup({required this.title, required this.range, this.tooltip});
+  ColumnGroup({required this.title, required this.range});
 
-  final String title;
+  ColumnGroup.fromText({
+    required String title,
+    required Range range,
+    String? tooltip,
+  }) : this(
+          title: maybeWrapWithTooltip(child: Text(title), tooltip: tooltip),
```

Comment:
instead of adding an additional constructor just add `maybeWrapWithTooltip` to the one line change in table.dart.

Reply:
I want to be able to pass widget to the constructor to create drop down for size type (shallow or retained).
So, we need two constructors: one that takes text and another that takes widget.